Dockerfile

```diff
@@ -29,10 +24,46 @@ RUN <<EOF
         autopoint \
         gettext \
         git \
-        libtool
+        libtool \
+        bison
     ln -s /usr/bin/libtoolize /usr/bin/libtool
+    apt-get clean
+    rm -rf /var/lib/apt/lists/*
 EOF
 
+FROM build_image AS nsenter
+
+ARG NSENTER_VERSION=v2.39.1
+ARG BUILD_DIR=/build
+ARG EXPORT_DIR=/export
+
+WORKDIR $BUILD_DIR
+RUN git clone -b $NSENTER_VERSION 'https://github.com/util-linux/util-linux.git'
+
+ARG BUILD_CACHE=build.cache
+RUN echo ac_cv_func_realloc_0_nonnull=yes >$BUILD_CACHE \
+    && echo ac_cv_func_malloc_0_nonnull=yes >>$BUILD_CACHE
```

Comment:
If `echo` works as in Bash, this can be shortened to `echo -e "ac_cv_func_realloc_0_nonnull=yes\nac_cv_func_malloc_0_nonnull=yes" >$BUILD_CACHE`

Reply:
This will need to change anyway for the final solution - keeping it for now